sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/AbstractSentryWebFilter.java

```diff
@@ -62,21 +63,23 @@ public AbstractSentryWebFilter(final @NotNull IHub hub) {
 
   protected void doFinally(
       final @NotNull ServerWebExchange serverWebExchange,
-      final @NotNull IHub requestHub,
+      final @NotNull IScopes requestHub,
```

Comment:
Should we rename this to something like `requestScopes`?

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366